### ISSUE TITLE
[FE7] Lyn Chapter 6 Doors (Reward+Thief shuffle)

### ISSUE THREAD

**Issue by Exuno** [lushen124/Universal-FE-Randomizer#366]
This map requires opening two doors - Matthew / whoever gets shuffled into his slot seems to start with one Door Key (if they're not a thief, I assume?), which also resolves the Hector chapter 11 issue, but I also got a Fimbulvetr instead of the Door Key from the Village. One enemy also had a door key, but it wasn't a drop (and obviously I didn't have a thief), and I don't think I could manipulate the AI to use it?

It should be simple enough to remove that village from the reward shuffle pool just the make the map safer overall, I would think?

**Comment by lushen124** [comment]
Yeah, seems like the simplest fix would be to keep the Door Key in this instance from the village. Alternatively, spawn Matthew with two keys in this specific instance. I could go either way on it.
